app/src/main/java/org/luxoft/sdl_core/BleCentralService.java

```diff
@@ -118,6 +129,20 @@ public void run() {
 
                         break;
 
+                    case ON_BLE_PERIPHERAL_DISCONNECT:
+                        if (mNativeBleAdapterThread != null) {
+                            mNativeBleAdapterThread.ReadMessageFromNative(mCallback);
+
+                            BluetoothPeripheral peripherial = mBluetoothHandler.getPeripheral();
+                            if(peripherial != null) {
+                                String ctrlMEssage = "{{ { \"action\": \"ON_DEVICE_DISCONNECTED\", \"params\": { " +
```

Comment:
Same as above

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83